src/Symfony/Component/Mailer/Bridge/Azure/Transport/AzureApiTransport.php

```diff
@@ -28,37 +28,32 @@ final class AzureApiTransport extends AbstractApiTransport
 {
     private const HOST = '%s.communication.azure.com';
 
-    /**
-     * User Access Key from Azure Communication Service (Primary or Secondary key).
-     */
-    private string $key;
-
-    /**
-     * The endpoint API URL to which to POST emails to Azure
-     * https://{acsResourceName}.communication.azure.com/.
-     */
-    private string $resourceName;
-
-    /**
-     * The version of API to invoke.
-     */
-    private string $apiVersion;
-
-    /**
-     * Indicates whether user engagement tracking should be disabled.
-     */
-    private bool $disableTracking;
-
-    public function __construct(string $key, string $resourceName, bool $disableTracking = false, string $apiVersion = '2023-03-31', HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null)
-    {
+    public function __construct(
+        /**
+         * User Access Key from Azure Communication Service (Primary or Secondary key).
+         */
+        private string $key,
+        /**
+         * The endpoint API URL to which to POST emails to Azure
+         * https://{acsResourceName}.communication.azure.com/.
+         */
+        private string $resourceName,
+        /**
+         * Indicates whether user engagement tracking should be disabled.
+         */
+        private bool $disableTracking = false,
+        /**
+         * The version of API to invoke.
+         */
+        private string $apiVersion = '2023-03-31',
+        HttpClientInterface $client = null,
+        EventDispatcherInterface $dispatcher = null,
+        LoggerInterface $logger = null,
+    ) {
```

Comment:
```suggestion
    /**
     * @param string $key              User Access Key from Azure Communication Service (Primary or Secondary key).
     * @param string $resourceName    The endpoint API URL to which to POST emails to Azure (https://{acsResourceName}.communication.azure.com/).
     * @param bool   $disableTracking Indicates whether user engagement tracking should be disabled.
     * @param string $apiVersion      The version of API to invoke.
     */
    public function __construct(
        private string $key,
        private string $resourceName,
        private bool $disableTracking = false,
        private string $apiVersion = '2023-03-31',
        HttpClientInterface $client = null,
        EventDispatcherInterface $dispatcher = null,
        LoggerInterface $logger = null,
    ) {
```

Reply:
I think, as these are properties now, I cannot use `@param` annotation. Because of this, we don't use them in 
* https://github.com/symfony/symfony/pull/51971/files

cc @GromNaN @alexandre-daubois

Reply:
@derrabus' version look correct to me

Reply:
Indeed, Alexander's suggestion works (in PhpStorm at least). Well, I think I have a PR to update 😄

Reply:
Don't you want to update as suggested @OskarStark? Why?